app/frontend/src/styles/application.scss

```diff
@@ -23,6 +23,7 @@ $govuk-new-link-styles: true;
 @import 'application/full-width-mast';
 @import 'application/govuk-notification-banner';
 @import 'application/govuk-phase-banner';
+@import 'application/hiring-staff-landing-page';
```

Comment:
whilst the current 'structure' of the css folders/files probably has limited shelf life with plans i have - could this file go in the publishers (hiring staff) folder to keep consistency with the current status quo? and then the file name is obviously just `landing-page`